lightkurve/utils.py

```diff
@@ -454,3 +455,43 @@ def wrapper(*args, **kwargs):
             finally:
                 sys.stdout = old_out
     return wrapper
+
+def detect_filetype(header):
+    """
+    Detects filetype of a given header.
+
+    This function will detect the file type by looking at both the TELESCOP and
+    CREATOR keywords in the first extension of the FITS header.
+    If the file is not recognized as a Kepler or TESS data product,
+    `None` will be returned.
+
+    Parameters
+    ----------
+    header :
+        The `header` for the first index of a fits file's hdulist.
+
+    Returns
+    -------
+    filetype : str or None
+        A string describing the detected filetype. If the filetype is not
+        recognized, `None` will be returned.
```

Comment:
You may want to list here, or earlier in the docstring, exactly which four strings this function currently returns.  This is quite obvious from the code, but the sphinx docs page for this function will not show the code by default.